lib/ruby_lsp/requests/semantic_highlighting.rb

```diff
@@ -115,31 +124,41 @@ def visit_call(node)
 
       sig { override.params(node: SyntaxTree::Command).void }
       def visit_command(node)
+        return super unless visible?(node, @range)
```

Comment:
I feel `visible?` should just read `@range` directly and only take `node`, like `visible?(node)`. 

Passing an instance variable to a method that's accessible to it seems to be unnecessary, at least in this case.

Reply:
I agree, but unfortunately if we do that we'll be forced to declare `@range` in `BaseRequest` because Sorbet complains that it's undeclared. Since not every request depends on a range, I ended up just passing it via arguments.

Reply:
I see 👍